src/software/geom/voronoi_util.cpp

```diff
@@ -0,0 +1,96 @@
+#include <g3log/loglevels.hpp>
+#include <g3log/g3log.hpp>
+#include "software/geom/voronoi_util.h"
+#include "software/geom/util.h"
+
```

Comment:
We should make sure to write unit tests for all the functions in this file.

Reply:
done